packages/nimble-components/src/table/tests/table-column-sizing.spec.ts

```diff
@@ -164,7 +190,7 @@ describe('Table Column Sizing', () => {
                 column2ExpectedRenderedWidth: 100
             }
         ];
-        const focused: string[] = [];
+        const focused: string[] = ['both columns use pixelWidth'];
```

Comment:
I'm guessing you aren't planning on checking this code in with only one of these tests running?